### PR TITLE
Taiga #19 - Improve PEM checking

### PR DESCRIPTION
**Pull request by anewmanRH**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 50d3075b9c5c9ce3041aafd26515e94511e0377a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 719815c177eef143b7c8b65cea407da738009c85:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 579e2d98873e1dfc6c640e78d284c0e2daf938af:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/test_appliance.py [hunk 1]
```diff
@@ -148,43 +147,18 @@ def test_cpu_total(ssh_client):
...
-            md5_of_file = re.split(r"\s+", md5_of_file, 1)[0]
-            soft_assert(given_md5 == md5_of_file, "md5 of file %s differs" % filename)
+    """Test whether the required product certificates are present."""
+
+    known_certs = ["/etc/rhsm/ca/redhat-uep.pem",
+    "/etc/rhsm/ca/candlepin-stage.pem", "/etc/pki/product-default/69.pem",
+    "/etc/pki/product/167.pem", "/etc/pki/product/201.pem"]
+
+    for cert in known_certs:
+        cert_path_vaild = ssh_client.run_command("test -f '%s'" % cert)[0] == 0
```
Comment (psav) [inline cfme/tests/test_appliance.py]: let's replace the ```%s``` with format like so
```python
    cert_path_vaild = ssh_client.run_command("test -f '{}'".format(cert))[0] == 0
```

same for the one below too

**Comment by dajoRH** [comment]
Lint report for commit a60b4d5efaf77c12524cc03b5fa9aff6888166a2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cec2a5e0459f84fd021531d12a07aac465e0c163:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**